plugins/Monolog/config/config.php

```diff
@@ -6,9 +6,11 @@
 
 return array(
 
-    'Psr\Log\LoggerInterface' => DI\object('Monolog\Logger')
+    'Monolog\Logger' => DI\object('Monolog\Logger')
```

Comment:
It's a detail but FYI this is the same as `'Monolog\Logger' => DI\object()` (no need to fix)